src/engine/Settings.cs

```diff
@@ -214,6 +214,430 @@ private Settings()
+    ///   The Db value to be added to the gui audio bus
+    /// </summary>
+    public SettingValue<float> VolumeGUI { get; set; } = new SettingValue<float>(0.0f);
+
+    /// <summary>
+    ///   If true gui audio bus is muted
+    /// </summary>
+    public SettingValue<bool> VolumeGUIMuted { get; set; } = new SettingValue<bool>(false);
+
+    // Performance Properties
+
+    /// <summary>
+    ///   If this is over 0 then this limits how often compound clouds
+    ///   are updated. The default value of 0.020 at 60 FPS makes
+    ///   every other frame not update the clouds.
+    /// </summary>
+    /// <remarks>
+    ///   <para>
+    ///     This should be made user configurable for different
+    ///     computers. The choises should probably be:
+    ///     0.0f, 0.020f, 0.040f, 0.1f, 0.25f
+    ///   </para>
+    /// </remarks>
+    public SettingValue<float> CloudUpdateInterval { get; set; } = new SettingValue<float>(0.040f);
+
+    /// <summary>
+    ///   This can be freely adjusted to adjust the performance The
+    ///   higher this value is the smaller the size of the simulated
+    ///   cloud is and the performance is better.
+    /// </summary>
+    public SettingValue<int> CloudResolution { get; set; } = new SettingValue<int>(2);
+
+    /// <summary>
+    ///   If true an auto-evo run is started during gameplay,
+    ///   taking up one of the background threads.
+    /// </summary>
+    public SettingValue<bool> RunAutoEvoDuringGamePlay { get; set; } = new SettingValue<bool>(true);
+
+    // Misc Properties
+
+    /// <summary>
+    ///   When true the main intro is played
+    /// </summary>
+    public SettingValue<bool> PlayIntroVideo { get; set; } = new SettingValue<bool>(true);
+
+    /// <summary>
+    ///   When true the microbe intro is played on new game
+    /// </summary>
+    public SettingValue<bool> PlayMicrobeIntroVideo { get; set; } = new SettingValue<bool>(true);
+
+    /// <summary>
+    ///   If false auto saving will be disabled
+    /// </summary>
+    public SettingValue<bool> AutoSaveEnabled { get; set; } = new SettingValue<bool>(true);
+
+    /// <summary>
+    ///   Number of auto saves to keep
+    /// </summary>
+    public SettingValue<int> MaxAutoSaves { get; set; } = new SettingValue<int>(5);
+
+    /// <summary>
+    ///   Number of quick saves to keep
+    /// </summary>
+    public SettingValue<int> MaxQuickSaves { get; set; } = new SettingValue<int>(5);
+
+    /// <summary>
+    ///   Saves the current settings by writing them to the settings configuration file.
+    ///   Show tutorial messages
+    /// </summary>
+    public SettingValue<bool> TutorialsEnabled { get; set; } = new SettingValue<bool>(true);
+
+    /// <summary>
+    ///   When true cheats are enabled
+    /// </summary>
+    public SettingValue<bool> CheatsEnabled { get; set; } = new SettingValue<bool>(false);
+
+    public SettingValue<bool> OpenScreenshotFolder {get; set; } = new SettingValue<bool>(false);
```

Comment:
What happend here?
Looks like you CTRL+C, V'd half of the class. You declare two `Settings` classes in the `Settings.cs`
```suggestion

```

Reply:
I think that is a bad merge sorry I'm lookin in today sorry for that